pkg/kubectl/cmd/taint.go

```diff
@@ -365,24 +378,21 @@ func validateNoTaintOverwrites(obj runtime.Object, taints []v1.Taint) error {
 	return utilerrors.NewAggregate(allErrs)
 }
 
-// updateTaints updates taints of obj
-func (o TaintOptions) updateTaints(obj runtime.Object) error {
+// updateTaints validates the overwriting of existing taints and reorganizes them on a node.
```

Comment:
Please be more verbose. "updateTaints adds/removes taints indicated by o (I hate this pattern by the way), while disallowing adding a taint if it would replace existing one if o.overwrite is false.". Or something like that - some native speaker with less than 12 hours on the clock today might have a better suggestion: @timothysc @davidopp